Added game settings

trudeaua opened the pull request:
### New stuff
- Added game settings (audio and video) to the main menu and pause menu
    - Only audio settings do anything noticeable at this time, we have to define some stuff for video quality
- Added controls with proper PS4 controller icons to the menus
- Menus remember which button was clicked so it's highlighted when you navigate backwards

camvr commented on Cosmic Quest - Order and Chaos/Assets/Scripts/Entity/Player/PlayerStatsController.cs:
```diff
@@ -177,7 +173,7 @@ protected virtual void OnMenuClose(InputValue value)
             return;
         }
-        if (PauseMenuController.Instance.IsPaused)
+        if (PauseMenuController.Instance.IsPaused && PauseMenuController.Instance.IsAtRoot())
```
I almost feel like we should have a different component to handle all of this user input stuff, but oh well... Not a big problem ultimately, maybe we'll look into changing that when we clean up code for the final implementation

camvr commented on Cosmic Quest - Order and Chaos/Assets/Scripts/Menu/MenuController.cs:
```diff
@@ -88,4 +138,182 @@ protected virtual void SetPlayerRoot(MultiplayerEventSystem eventSystem)
+    /// </summary>
+    protected void FindCameraAndMusic()
+    {
+        mainCamera = FindObjectOfType<Camera>();
```
Maybe use `Camera.main` for this instead? Even though we only have one camera at the moment, when we do a minimap this might end up grabbing the wrong camera

trudeaua commented:
@camvr changes addressed in latest commits. Also added a loading screen that overlays when loading a level and fixed that issue where the dialogue broke if players spammed the button to load a level
